server/tests/socialAuth.test.js

```diff
@@ -9,7 +9,7 @@ chai.use(chaiHttp);
 
 describe('Test for Passport Authentication', () => {
   it('should return 200 for successful authentication', (done) => {
-    chai.request(app).get('/api/auth/mock').end((req, res) => {
+    chai.request(app).get('/api/auth/mocks').end((req, res) => {
```

Comment:
I do not understand why you are changing the auth/mock controller to mocks instead of mock. I don't also think this has anything to do with the create rating to the article. can you return it back to its default which is mock without an s. Thank you

Reply:
Sorry about that, I guess I missed it

Reply:
I have implemented your feedback @Oluwafayokemi, please review.